packages/cli/templates/default-app/packages/app/src/App.tsx

```diff
@@ -0,0 +1,42 @@
+import { CssBaseline, makeStyles, ThemeProvider } from '@material-ui/core';
+import { BackstageTheme, createApp } from '@backstage/core';
+import React, { FC } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import * as plugins from './plugins';
+
+const useStyles = makeStyles(theme => ({
+  '@global': {
```

Comment:
i wonder if this could be in core instead.

Reply:
Yeah plenty of opportunities for moving things from the example app. Gonna be important for keeping sync too.